apps/sms/views/shared/style/root.css

```diff
@@ -12,7 +12,7 @@ body {
   font-size: 1.6rem;
   line-height: 2.2rem;
   vertical-align: middle;
-  background-color: #FFF;
+  background: var(--header-background);
```

Comment:
__question:__ wondering if we should use ```--highlight-color``` variable instead here? In Gaia Theme this variable is in ```Brand Color``` section, the name is confusing to me though.